ci/jenkins/test.sh

```diff
@@ -68,7 +69,8 @@ Run K8s e2e community tests (Conformance & Network Policy) or Antrea e2e tests o
         --testbed-type           The testbed type to run tests. It can be flexible-ipam, jumper or legacy.
         --ip-mode                IP mode for flexible-ipam e2e test. Default is $DEFAULT_IP_MODE. It can also be ipv6 or ds.
         --win-image-node         Name of the windows image node in containerd cluster. Images are built by docker on this node.
-        --kind-cluster-name      Name of the kind Cluster." 
+        --kind-cluster-name      Name of the kind Cluster
+        --enable-ipv6            Enable the ipv6 version for the kind conformanc test." 
```

Comment:
Is this parameter `--enable-ipv6` only for kind? I feel it's confusing as we already had `--ip-mode`.

Reply:
Yes it's only for ipv6 conformance test, I will reuse existing one.